tests/pages/freestyle-guide.js

```diff
@@ -1,12 +1,15 @@
 import PageObject, {
   collection,
   clickable,
+  clickOnText,
   text,
-  visitable
+  visitable,
+  contains
 } from 'ember-cli-page-object';
 
 export default PageObject.create({
-  visit: visitable('/?s=Foo%20Things'),
+  visit: visitable('/'),
+  visitFooThings: visitable('/?s=Foo%20Things'),
```

Comment:
Perhaps call this `visitFooThingsSection` in anticipation of future tests navigating elsewhere?